test/Target.gd

```diff
@@ -0,0 +1,17 @@
+extends Area3D
+
+
+var health = 1000
+
+
+func _ready() -> void:
+	_update()
+
+
+func hit():
```

Comment:
Not sure if I understood, but other classes call (A Collider).hit()?
Does that mean that the hit function is attached / coupled with the colliders instead of the associated class of that collider?